src/App.js

```diff
@@ -1,16 +1,86 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
+import Todo from './components/TodoComponents/Todo'
+import TodoForm from './components/TodoComponents/TodoForm'
+import './components/TodoComponents/Todo.css'
+
+const listData = []
+
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(){
+    super();
+    this.state = {
+      list: listData,
+      inputText: '',
+    };
+  }
+  handleChange = event => {
+    this.setState ({
+      [event.target.name]: event.target.value
+    });
+    console.log(event.target.name);
+  };
+  
+  addItem = ev => {
+    ev.preventDefault();
+    this.setState ({
+      list: [
+        ...this.state.list,
+        {task: this.state.inputText, id: Date.now(),completed: 'false' }
```

Comment:
The only danger in using Date.now for ids is if you have an application that makes many items at once you can end up with non-unique ids.  This works perfectly for this application though.